mutex/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "temporal-mutex",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build",
+    "build.watch": "tsc --build --watch",
+    "lint": "eslint .",
+    "start": "ts-node src/worker.ts",
+    "start.watch": "nodemon src/worker.ts",
+    "test": "mocha ./lib/test/*.test.js",
+    "lock-workflow": "ts-node src/start-lock-workflow.ts",
+    "test-workflow": "ts-node src/start-test-workflow.ts"
+  },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node"
+    },
+    "ext": "ts",
+    "watch": [
+      "src"
+    ]
+  },
+  "dependencies": {
+    "@temporalio/activity": "^1.5.2",
+    "@temporalio/client": "^1.5.2",
+    "@temporalio/worker": "^1.5.2",
+    "@temporalio/workflow": "^1.5.2",
+    "uuid": "8.x"
```

Comment:
nit: can we use `crypto.randomUUID` instead of adding this dependency?

Reply:
Will switch to nanoid to be consistent with other samples. Nanoid is nice because it generates a much shorter id than a uuidv4, which is better for an example app.